src/components/admin/form-components/MaxLenTextField.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { capFirstLetter } from 'lib/utilities';
+import TextField from 'material-ui/TextField';
+
+export default class MaxLenTextField extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onChange = this.onChange.bind(this);
+  }
+
+  onChange(event) {
```

Comment:
Should be inlineable as well, also remember if you use arrow functions in the constructor it automatically binds this

Reply:
Or maybe I'm going too far with this and it's okay in small components like this? What do you think?

Reply:
No, I agree - TODO